### PR TITLE
I18n updates

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by manstis** [comment]
TaskViewDetailsImp.. should the change include "high", "medium" and "low"?

**Comment by cristianonicolai** [comment]
@manstis not sure what you mean exactly. High, medium and low are in the constants file already, used by another screen/presenter.

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskdetails/TaskDetailsViewImpl.java [hunk 1]
```diff
@@ -95,14 +95,14 @@
     @Inject
     private PlaceManager placeManager;
 
-    private String[] priorities = { "0 - High", "1", "2", "3", "4", "5 - Medium", "6", "7", "8", "9", "10 - Low" };
+    private String[] priorities = { "0 - " + constants.High(), "1", "2", "3", "4", "5 - " + constants.Medium(), "6", "7", "8", "9", "10 - " + constants.Medium() };
```
Comment (psiroky) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskdetails/TaskDetailsViewImpl.java]: I am not sure if this is what @manstis meant, but the last constant here should be `constants.Low()`.

**Comment by manstis** [comment]
Petr is right... sorry on phone and unable to comment on lines :-(

**Comment by cristianonicolai** [comment]
that makes sense :) thanks guys.

**Comment by cristianonicolai** [comment]
done :)

**Comment by manstis** [comment]
Any chance you can squash the commits?

**Comment by cristianonicolai** [comment]
finally ;)

**Comment by manstis** [comment]
+1 to merge.

**Pull request merged**